client/my-sites/domains/domain-management/list/domain-row.jsx

```diff
@@ -284,7 +284,7 @@ class DomainRow extends PureComponent {
 
 		return (
 			<a href="#" onClick={ this.addEmailClick }>
-				{ translate( 'Add +', { context: 'Button label' } ) }
+				{ translate( '+ Add', { context: 'Button label' } ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.